sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/ItemBatchOperation.java

```diff
@@ -66,7 +68,7 @@ JsonSerializable serializeOperation() {
         if (this.getItemInternal() != null) {
             if (this.getOperationType() == CosmosItemOperationType.PATCH) {
                 jsonSerializable.set(BatchRequestResponseConstants.FIELD_RESOURCE_BODY,
-                    PatchUtil.serializableBatchPatchOperation(this.getItemInternal()));
+                    PatchUtil.serializableBatchPatchOperation(new PatchSpec((CosmosPatchOperations)this.getItemInternal(), this.getRequestOptions())));
```

Comment:
For all the occurrence of PatchSpec, I can see we create an instance just before calling the utility class. Why can't we call it simply with item and request option? @moderakh - correct me if I am wrong, is it required to create a new instance?

Reply:
@rakkuma you are right. I don't see why you would need to wrap it.